### ISSUE TITLE
Inbox – Starting a Convo

### ISSUE THREAD

**Issue by devanshah1991** [hindsight-io/connect#128]
If you click on someones profiles and click “message,” it will start a permanent inbox message convo for both users even though neither user has said anything yet. Is it easy to get rid of this so that a message only shows up in the inbox if at least one message has been sent?

**Comment by dax0** [comment]
Good point but will have to table this till I'm able to rewrite the messaging portion
